### ISSUE TITLE
CP starts fieldwork while weeder is still unfolding

### ISSUE THREAD

**Issue by SilverWheelz** [Courseplay/Courseplay_FS22#124]
I've had this occur only with the TS 1520/M3 Weeder so far. Unlike other implements you can lower this weeder without unfolding. So I assume CP only checks if the implement has been lowered before starting since most implements can't be lowered if they are still folded/unfolding. This causes CP to start before the weeder has been completely unfolded causing it to miss the first few feet of land.

**Comment by pvaiko** [comment]
Just out of curiosity, does the built-in helper wait for the unfold?

**Comment by dominiczeth** [comment]
To answer here: yes default helper waits.

**Comment by DHempelmann** [comment]
It is the same with the Panther (sugarbeet). It just lowers the cutter, but does not unfold the mashine itself.

**Comment by pull2play** [comment]
Same issue with the 24.5m roller.

**Comment by Tensuko** [comment]
> It is the same with the Panther (sugarbeet). It just lowers the cutter, but does not unfold the mashine itself.

This is already in another Issue.


How is the status of this in the latest release (currently 7.0.0.16) ?

**Comment by schuelli238** [comment]
Still the same behavior. Weeder and sowing machines.

**Comment by Tensuko** [comment]
This might work better now.
We still have on our list a general controller for such stuff, but not sure when this will happen.

**Comment by pvaiko** [comment]
We have to ask Stefan (Giants) why `getCanAIFieldWorkerContinueWork()` returns true when the weeder is lowered but not yet unfolded.

**Comment by pvaiko** [comment]
> To answer here: yes default helper waits.

Nope. The answer is no, the default helper does not wait either. This seems to be a bug in the game.

**Comment by Tensuko** [comment]
Will be fixed in a later Gamepatch (not 1.3 but should be in 1.4).
